superset-frontend/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -159,9 +171,14 @@ class ChartHolder extends React.Component {
     });
   }
 
+  handleFullSize() {
```

Comment:
Can we rename to `handleToggleFullSize` since that seems to be what's happening here?

Reply:
Renamed c0526bd607a84ebc9e32f7d4676af14a06a2624b